packages/builder/src/components/backend/TableNavigator/TableDataImport.svelte

```diff
@@ -57,23 +68,22 @@
 
   async function handleFile(evt) {
     const fileArray = Array.from(evt.target.files)
-    const filesToProcess = fileArray.map(({ name, path, size }) => ({
-      name,
-      path,
-      size,
-    }))
-
-    if (filesToProcess.some(file => file.size >= FILE_SIZE_LIMIT)) {
+    if (fileArray.some(file => file.size >= FILE_SIZE_LIMIT)) {
       notifier.danger(
         `Files cannot exceed ${FILE_SIZE_LIMIT /
           BYTES_IN_MB}MB. Please try again with smaller files.`
       )
       return
     }
 
-    files = filesToProcess
-
-    await validateCSV()
+    // Read CSV as plain text to upload alongside schema
+    let reader = new FileReader()
```

Comment:
Is this something we want to do in the browser? The primary reason that this was done using paths/electron originally was to prevent the entire CSV being read into memory and sent over the wire.

Reply:
> 
> 
> Interesting images lol

Haha I downloaded a folder of stock images, didn't even really look at them but yea I see your point...

Reply:
About performance - in my testing the overhead is pretty trivial. Parsing a CSV of even 10K rows is instant and the amount sent over the wire is capped since we reject files over 1MB anyway so that won't be an issue. I'll try with as big a CSV as I can find less than 1MB and see if I can notice any sort of parsing time.

Reply:
So i tried with a 840KB CSV - 11751 rows with 6 fields each and I still can't notive any sort of parsing delay at all so I think it's pretty safe to do in the browser, considering we have the 1MB limit. Reading a plain text file less than 1MB in to memory shouldn't be an issue on even an old mobile CPU I would say.

Reply:
Sounds good - worth the tradeoff in terms of dev speed in the browser then 👍